src/sui-link/index.jsx

```diff
@@ -24,7 +25,12 @@ export default class Link extends React.Component{
     );
 
     return (
-        <a className={customclass} href={this.props.url} title={this.props.title} target={this.props.target}>
+        <a className={customclass}
+           href={this.props.url}
+           title={this.props.title}
+           target={this.props.target}
+           onClick={this.props.handleClick}
```

Comment:
What about if handleClick is not defined? what's the html returned? will it has an empty handleClick property?